external/fv3fit/fv3fit/reservoir/reservoir.py

```diff
@@ -19,12 +19,13 @@ def _f(d):
     return _f
 
 
-def _random_uniform_sparse_matrix(m, n, sparsity, min=0, max=1):
+def _random_uniform_sparse_matrix(m, n, sparsity, min=0, max=1, type="csc"):
```

Comment:
Switched to CSC sparse matrix to improve speed.  If we only ever have a single reservoir and the size isn't too large, just using regular arrays speeds up multiplication by another order of magnitude.